src/Microsoft.Framework.PackageManager/Restore/RemoteWalkProvider.cs

```diff
@@ -16,10 +16,10 @@ public class RemoteWalkProvider : IWalkProvider
     {
         private readonly IPackageFeed _source;
 
-        public RemoteWalkProvider(IPackageFeed source, bool isHttp)
+        public RemoteWalkProvider(IPackageFeed source)
         {
             _source = source;
-            IsHttp = isHttp;
+            IsHttp = IsHttpSource(source);
```

Comment:
Moving initialization of `IsHttp` here can simplify extraction of shared code (`PackageSourceUtils.CreatePackageFeed()`). Moreover, the `source` argument already contains `IsHttp` information, so passing `isHttp` introduces duplicated info.